emily_cron/app/services/deposit_processor.py

```diff
@@ -88,17 +88,18 @@ def process_rbf_transactions(
             logger.info(
                 f"Confirmed transaction {confirmed_txid_in_group} found for RBF group {rbf_key}"
             )
-            # Mark all collected unconfirmed transactions as FAILED
+            # Mark all collected unconfirmed transactions as RBF'd
             for tx in unconfirmed_txs_in_group:
                 logger.info(
-                    f"Marking RBF'd transaction {tx.bitcoin_txid} as FAILED (replaced by confirmed tx {confirmed_txid_in_group})"
+                    f"Marking RBF'd transaction {tx.bitcoin_txid} as RBF'd (replaced by confirmed tx {confirmed_txid_in_group})"
```

Comment:
Well, the status will be `RBF`, so saying that we'll mark it as `RBF'd` seems wrong. On the other hand, it reads better the way that it is now. So 🤷🏽 
```suggestion
                    f"Marking RBF'd transaction {tx.bitcoin_txid} as RBF (replaced by confirmed tx {confirmed_txid_in_group})"
```